packages/grid/src/vaadin-grid-helpers.js

```diff
@@ -105,6 +104,35 @@ export function updateRowStates(row, states, appendValue) {
   });
 }
 
+/**
+ * @param {!HTMLElement} row
+ * @param {Object} states
+ */
+export function updateStringRowStates(row, states) {
```

Comment:
I'm really not sure about the function name... 
It turned out that the original updateRowStates works great for `boolean` states (even, odd, expanded, selected etc.) however it fails when state is not only boolean but a string (like dragover).

The original code addressed it by the appendValue parameter, but it didn't work that well. When I started to make chagnes to updateRowStates, there were so many of them, that I decided that for the readibility sake it would be better to separate it to a new method....

What the name... is weird...

Reply:
It is fine by me to go with this naming if we also rename `updateRowStates` to `updateBooleanRowStates`.